src/app/view-item/view-item.component.ts

```diff
@@ -1,5 +1,12 @@
-import { Component } from '@angular/core';
-import { Item } from '../model';
+import { Component, Input } from '@angular/core';
+
+export interface TodoItem {
+  list: string;
+  id: number;
+  title: string;
+  description?: string;
+  completed: boolean;
+}
```

Comment:
This interface has been moved and renamed inside the data service. Use that instead of redefining it. (Probably you did not rebase after data-service has been accepted)